velox/vector/arrow/Bridge.cpp

```diff
@@ -1446,6 +1450,10 @@ void exportToArrow(
           maps.getNullCount());
       exportToArrow(rows, *child, options);
       child->name = "entries";
+      // Map data should be a non-nullable struct type
+      clearNullableFlag(child->flags);
+      // Map data key type should be a non-nullable
```

Comment:
nit: ".. should be non-nullable."

(please also use period after comment sentences)

Reply:
DONE.